src/software/ai/hl/stp/play/timeout_play/timeout_play_fsm.cpp

```diff
@@ -0,0 +1,41 @@
+#include "software/ai/hl/stp/play/timeout_play/timeout_play_fsm.h"
+
+#include "software/ai/hl/stp/tactic/move/move_tactic.h"
+
+PriorityTacticVector TimeoutPlayFSM::makeTimeoutFormation(WorldPtr world)
+{
+    TacticVector tactic = {};
```

Comment:
```suggestion
    TacticVector tactics = {};
```
better reflects that the fact that multiple tactics are stored in this variable here

Reply:
done.